spec/services/calculator_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe Calculator, type: :service do
```

Comment:
There is no type `service`. Just leave the type away. I had the same issue once: https://stackoverflow.com/questions/31921486/rspec-most-accurate-type-for-a-service-test
I would be happy for an upvote :-P